app/javascript/src/plugins/shareMenuPlugin.jsx

```diff
@@ -24,18 +21,7 @@ const WindowTopBarShareMenu = (props) => (
   />
 )
 
-/**
- *
- * @param theme
- * @returns {{ctrlBtn: {margin: (number|string)}}}
- */
-const styles = theme => ({
```

Comment:
Find out if we need this style...

Reply:
This style can provide a class to Mirador here: https://github.com/ProjectMirador/mirador/blob/mui6/src/components/WindowTopMenuButton.js#L34

Visually, this button's background when selected looks to match the other buttons without this class.
So it seems on the surface that it isn't needed. 

<img width="1493" alt="Screenshot 2025-02-12 at 5 40 15 PM" src="https://github.com/user-attachments/assets/2edd66a9-0604-427e-852b-78e00265a7bb" />
<img width="1500" alt="Screenshot 2025-02-12 at 5 40 25 PM" src="https://github.com/user-attachments/assets/e0dc970f-9659-4bd9-835c-8c5948aba513" />